mmdeploy/backend/onnxruntime/wrapper.py

```diff
@@ -61,7 +61,7 @@ def __init__(self,
         self.io_binding = sess.io_binding()
         self.device_id = device_id
         self.is_cuda_available = is_cuda_available
```

Comment:
may remove this line.

Reply:
@sanjaypavo Hi, could you remove the unused attribute `is_cuda_available`?

Reply:
> @sanjaypavo Hi, could you remove the unused attribute `is_cuda_available`?

@RunningLeon Hi, remove the attribute.